index.js

```diff
@@ -2,8 +2,10 @@
 const fetch = require('node-fetch');
 const AbortController = require('abort-controller');
 
+const TEN_MEGABYTES = 10240 * 1024;
+
 if (!global.fetch) {
-	global.fetch = fetch;
+	global.fetch = options => fetch(options.url, Object.assign(options, {highWaterMark: options.highWaterMark || TEN_MEGABYTES}));
```

Comment:
@sholladay I'm not sure whether there is a cleaner way to write this, but I can confirm it works now:

```js
test('default highWaterMark is 10MB', async t => {
	const response = await ky('https://httpbin.org/json');
	t.is(response.highWaterMark, 10485760);
});

test('can override the highWaterMark', async t => {
	const response = await ky('https://httpbin.org/json', {highWaterMark: 1024 * 1024});
	t.is(response.highWaterMark, 1048576);
});
```

Reply:
I'm away from my laptop at the moment, but I'm happy to help tomorrow. Basically, it should use object spread the way it used to be in your previous commit, but the default options need to come before the user's options object. When using object spread, the order matters, with later options overriding earlier ones. Does that make sense?

Reply:
@xxczaki Basically this -

```suggestion
	global.fetch = (url, options) => fetch(url, {highWaterMark: TEN_MEGABYTES, ...options});
```

Reply:
@sholladay The problem is, in `node-fetch@3.0.0-beta.5`, the default `highWaterMark` is undefined and not 16384. This was fixed and therefore we will be able to use the spread syntax with `3.0.0-beta.6`, which should come out in a few days max. 

So I think we should wait for the next release ;)